typescript/packages/wallets/sui/src/sendSui.plugin.ts

```diff
@@ -0,0 +1,43 @@
+import { Chain, PluginBase, createTool } from "@goat-sdk/core";
+import { TransactionBlock } from "@mysten/sui.js/transactions";
+import { z } from "zod";
+import { SuiWalletClient } from "./SuiWalletClient";
+
+// Define the parameters schema for sending SUI
+const sendSUIParametersSchema = z.object({
+    to: z.string().describe("The recipient's address"),
+    amount: z.string().describe("The amount of SUI to send"),
+});
+
+// Implementation of the send method
+const sendSUIMethod = async (walletClient: SuiWalletClient, parameters: z.infer<typeof sendSUIParametersSchema>) => {
+    const { to, amount } = parameters;
+    const txb = new TransactionBlock();
```

Comment:
No, you use the Transaction object, check the docs here: https://docs.sui.io/guides/developer/sui-101/building-ptb

Reply:
The TransactionBlock class has been renamed to Transaction, and most methods and properties that previously used transactionBlock have been updated to use transaction instead.